metaflow/runner/deployer.py

```diff
@@ -397,6 +399,7 @@ def create(self, **kwargs) -> DeployedFlow:
             self.name = content.get("name")
             self.flow_name = content.get("flow_name")
             self.metadata = content.get("metadata")
+            self.additional_metadata = content.get("additional_metadata", {})
```

Comment:
minor nit - can you consider renaming this to `additional_meta` instead of `metadata` - so that it is apparent that this field has no relationship to metadata. also, if you can add a note above this line that this attribute is used in extensions - that will ensure that nobody considers this dead code in the future. otherwise it's good to go - please feel free to merge once the changes are in.